src/client.ts

```diff
@@ -876,6 +877,8 @@ export type ClientEventHandlerMap = {
     & HttpApiEventHandlerMap
     & BeaconEventHandlerMap;
 
+const UNSTABLE_MSC3824_ACTION_PARAM = new UnstableValue("action", "org.matrix.msc3824.action");
```

Comment:
we typically just call it something like `OIDC_ACTION_PARAM` so we can change the type to a `NamespacedValue` without breaking the SDK itself.

Reply:
I've renamed it `SSO_ACTION_PARAM`.